components/resolved_ts/src/advance.rs

```diff
@@ -6,6 +6,8 @@ use std::time::Duration;
 use collections::{HashMap, HashSet};
 use concurrency_manager::ConcurrencyManager;
 use engine_traits::KvEngine;
+#[cfg(feature = "failpoints")]
```

Comment:
It doesn't need to be guarded.

Reply:
`make clippy` will report a warning.

Reply:
If two feature cfgs are removed, then there should be no warnings.